<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Ultralight Browser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/ultralight.ico</normaloff>assets/ultralight.ico</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Tabs">
    <property name="title">
     <string>&amp;Tabs</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_Tabs"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Back"/>
   <addaction name="action_Forward"/>
   <addaction name="action_Reload"/>
   <addaction name="action_Stop"/>
   <addaction name="action_Home"/>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New tab</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close tab</string>
   </property>
  </action>
  <action name="action_Back">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="iconText">
    <string>&lt;</string>
   </property>
  </action>
  <action name="action_Forward">
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="iconText">
    <string>&gt;</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="iconText">
    <string> ⃞  </string>
   </property>
  </action>
  <action name="action_Reload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="iconText">
    <string>⭯</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Home">
   <property name="text">
    <string>Home</string>
   </property>
   <property name="iconText">
    <string>⌂</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
